### ISSUE TITLE
Deprecation: events.inherited-function-listeners

### ISSUE THREAD

**Issue by gandalfar** [DockYard/ember-in-viewport#191]
When upgrading my app, I've started seeing the following deprecation warning: `events.inherited-function-listeners`. I can't reproduce it inside dummy app in this repo, but it's present when I create a new ember app.

The full error is:

```
deprecate.js:122 DEPRECATION: You attempted to remove a function listener which did not exist on the instance, which means it was an inherited prototype listener, or you attempted to remove it before it was added. Prototype function listeners have been deprecated, and attempting to remove a non-existent function listener this will error in the future. [deprecation id: events.inherited-function-listeners] See https://emberjs.com/deprecations/v3.x#toc_events-inherited-function-listeners for more details.
        at logDeprecationStackTrace (http://localhost:4220/assets/vendor.js:46053:23)
        at HANDLERS.<computed> (http://localhost:4220/assets/vendor.js:46147:11)
        at raiseOnDeprecation (http://localhost:4220/assets/vendor.js:46080:11)
        at HANDLERS.<computed> (http://localhost:4220/assets/vendor.js:46147:11)
        at invoke (http://localhost:4220/assets/vendor.js:46159:11)
        at deprecate (http://localhost:4220/assets/vendor.js:46118:30)
        at Meta.pushListener (http://localhost:4220/assets/vendor.js:22772:35)
        at Meta.removeFromListeners (http://localhost:4220/assets/vendor.js:22705:12)
        at removeListener (http://localhost:4220/assets/vendor.js:23436:9)
```

Repository with a single component where I see this warning is here:
https://github.com/gandalfar/viewport-test

## Version
    "ember-source": "~3.8.1",
    "ember-in-viewport": "^3.5.1",

## Test Case
https://github.com/gandalfar/viewport-test

Component code is here:
https://github.com/gandalfar/viewport-test/blob/master/app/components/example-component.js

## Steps to reproduce
I open the page in Chrome 74 on macOS 10.13.6 and deprecation appears in Dev Tools.

**Comment by snewcomer** [comment]
@gandalfar This should be fixed in 3.5.2!

Ref #185

**Comment by gandalfar** [comment]
Works great. Thanks!
